packages/rest-client-jsonapi/src/services/utils/query/query.ts

```diff
@@ -83,11 +83,15 @@ export function filterToQueryParam(filterArr: Filter): string {
             .join("&"),
         )
       } else {
-        queries.push(
-          `filter[${key}]${operator === "empty" ? "" : `[${operator}]`}=${
-            operator !== "empty" ? encodeURIComponent(value) : null
-          }`,
-        )
+        if (operator === "empty") {
+          queries.push(`filter[${key}][$eq]=${null}`)
+        } else if (operator === "nempty") {
+          queries.push(`filter[${key}][$ne]=${null}`)
```

Comment:
any reason we can't use `$ne` as the key in `operatorOptions` in the `MuiFilter.tsx` file? is `$ne` jsonapi specific and `nempty` is more generic?

Reply:
`empty` and `nempty` use `$eq` and `$ne`. We need them separate on the frontend in order to have 'is empty, is not empty' and 'equals, does not equal' options in the dropdown (and to know when the value should be set to `null`).